load_testing/common_flows/flow_ial2_proofing.py

```diff
@@ -23,6 +23,7 @@ def do_ial2_proofing(context):
         "put",
         "/verify/doc_auth/welcome",
         "/verify/doc_auth/agreement",
+        "",
         {"authenticity_token": auth_token, },
```

Comment:
WDYT of calling with named arguments here?
```suggestion
        data={"authenticity_token": auth_token, },
```
Then we can remove the `""` for all of these right?

Reply:
that's the plan, there are tons of places where we can DRY up this code since a lot of it was simply C/Pd to expand functionality quickly. I'll do so in a follow up PR focused on just that.